### PR TITLE
[ADI] Add thread admin

### PR DESCRIPTION
**Pull request by JeremieSamson**

### CODE REVIEW

File: src/Admin/IdeasWorkshopThreadAdmin.php [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                'actions' => [
+                    'show' => [],
+                ],
+            ])
+        ;
+    }
+
+    protected function configureRoutes(RouteCollection $collection)
+    {
+        $collection->remove('edit');
```
Comment (ottaviano) [inline src/Admin/IdeasWorkshopThreadAdmin.php]: maybe `$collection->clearExcept('show')`?

File: src/Entity/IdeasWorkshop/Idea.php [hunk 2]
```diff
@@ -590,4 +590,9 @@ public function getReportType(): string
     {
         return ReportType::IDEAS_WORKSHOP_IDEA;
     }
+
+    public function __toString(): string
+    {
+        return $this->name;
```
Comment (ottaviano) [inline src/Entity/IdeasWorkshop/Idea.php]: `return (string) $this->name;`

File: templates/admin/ideas_workshop/thread/show_thread_comments.html.twig [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{% extends '@SonataAdmin/CRUD/base_show_field.html.twig' %}
+
+{% block field %}
+    {% for threadComment in object.comments %}
+        Auteur: {{ threadComment.author }}<br>
+        Date: {{ threadComment.createdAt|localizeddate('none', 'none', 'fr_FR', null, "d MMM Y HH:MM") }}<br>
+        Approuvé:
+        {% if threadComment.approved %}
+            <span class=" label label-success">{{ 'global.yes'|trans|lower }}</span>
```
Comment (ottaviano) [inline templates/admin/ideas_workshop/thread/show_thread_comments.html.twig]: `class=" label` -> `class="label`

File: templates/admin/ideas_workshop/thread/show_thread_comments.html.twig [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+        {% else %}
+            <span class=" label label-danger">{{ 'global.no'|trans|lower }}</span>
```
Comment (ottaviano) [inline templates/admin/ideas_workshop/thread/show_thread_comments.html.twig]: `class=" label` -> `class="label`

**Comment by ottaviano** [comment]
can you rollback composer.lock, please?

File: templates/admin/ideas_workshop/thread/show_thread_comments.html.twig (hunk 3 above)
Comment (JeremieSamson) [reply templates/admin/ideas_workshop/thread/show_thread_comments.html.twig]: thanks :)

File: templates/admin/ideas_workshop/thread/show_thread_comments.html.twig (hunk 4 above)
Comment (JeremieSamson) [reply templates/admin/ideas_workshop/thread/show_thread_comments.html.twig]: thanks :)

**Pull request merged**